vspec/model/constants.py

```diff
@@ -10,127 +10,159 @@
-    UNIX_TIMESTAMP_ARRAY = "UNIX Timestamp[]"
+class StringStyle(metaclass=VSSEnumMeta):
+    __config__ = {
+        'NONE': VSSConstant('none', 'none'),
+        'CAMEL_CASE': VSSConstant('camel case', 'camelCase'),
+        'CAMEL_BACK': VSSConstant('camel back', 'camelBack'),
+        'CAPITAL_CASE': VSSConstant('capital case', 'capitalcase'),
+        'CONST_CASE': VSSConstant('const case', 'constcase'),
+        'LOWER_CASE': VSSConstant('lower case', 'lowercase'),
+        'PASCAL_CASE': VSSConstant('pascal case', 'pascalcase'),
+        'SENTENCE_CASE': VSSConstant('sentence case', 'sentencecase'),
+        'SNAKE_CASE': VSSConstant('snake case', 'snakecase'),
+        'SPINAL_CASE': VSSConstant('spinal case', 'spinalcase'),
+        'TITLE_CASE': VSSConstant('title case', 'titlecase'),
+        'TRIM_CASE': VSSConstant('trim case', 'trimcase'),
+        'UPPER_CASE': VSSConstant('upper case', 'uppercase'),
+        'ALPHANUM_CASE': VSSConstant('alphanum case', 'alphanumcase'),
+    }
+
+
+class VSSType(metaclass=VSSEnumMeta):
+    __config__ = {
+        'BRANCH': VSSConstant('branch', 'branch'),
+        'RBRANCH': VSSConstant('rbranch', 'rbranch'),
+        'ATTRIBUTE': VSSConstant('attribute', 'attribute'),
+        'SENSOR': VSSConstant('sensor', 'sensor'),
+        'ACTUATOR': VSSConstant('actuator', 'actuator'),
+        'ELEMENT': VSSConstant('element', 'element'),
+    }
+
+
+class VSSDataType(metaclass=VSSEnumMeta):
+    __config__ = {
+        'INT8': VSSConstant('int8', 'int8'),
+        'UINT8': VSSConstant('uint8', 'uint8'),
+        'INT16': VSSConstant('int16', 'int16'),
+        'UINT16': VSSConstant('uint16', 'uint16'),
+        'INT32': VSSConstant('int32', 'int32'),
+        'UINT32': VSSConstant('uint32', 'uint32'),
+        'INT64': VSSConstant('int64', 'int64'),
+        'UINT64': VSSConstant('uint64', 'uint64'),
+        'BOOLEAN': VSSConstant('boolean', 'boolean'),
+        'FLOAT': VSSConstant('float', 'float'),
+        'DOUBLE': VSSConstant('double', 'double'),
+        'STRING': VSSConstant('string', 'string'),
+        'UNIX_TIMESTAMP': VSSConstant('UNIX Timestamp', 'UNIX Timestamp'),
+        'INT8_ARRAY': VSSConstant('int8[]', 'int8[]'),
+        'UINT8_ARRAY': VSSConstant('uint8[]', 'uint8[]'),
+        'INT16_ARRAY': VSSConstant('int16[]', 'int16[]'),
+        'UINT16_ARRAY': VSSConstant('uint16[]', 'uint16[]'),
+        'INT32_ARRAY': VSSConstant('int32[]', 'int32[]'),
+        'UINT32_ARRAY': VSSConstant('uint32[]', 'uint32[]'),
+        'INT64_ARRAY': VSSConstant('int64[]', 'int64[]'),
+        'UINT64_ARRAY': VSSConstant('uint64[]', 'uint64[]'),
+        'BOOLEAN_ARRAY': VSSConstant('boolean[]', 'boolean[]'),
+        'FLOAT_ARRAY': VSSConstant('float[]', 'float[]'),
+        'DOUBLE_ARRAY': VSSConstant('double[]', 'double[]'),
+        'STRING_ARRAY': VSSConstant('string[]', 'string[]'),
+        'UNIX_TIMESTAMP_ARRAY': VSSConstant('UNIX Timestamp[]', 'UNIX Timestamp[]'),
```

Comment:
There will be a merge conflict with https://github.com/GENIVI/vss-tools/pull/103, but that shall be easy to fix

Reply:
I can change whomever's PR passes last :)